src/Misc/layoutbin/run-helper.cmd

```diff
@@ -0,0 +1,34 @@
+@echo off
```

Comment:
hmmm wondering if we move the bin folder, will that disrupt run.cmd

Reply:
i guess the question is whether we are moving bin while run-helper.cmd is still executing

Reply:
should this file be powershell instead?

Reply:
(as long as the bin folder isnt the working directory)